lightning/kv/importer.go

```diff
@@ -34,8 +34,8 @@ import (
 )
 
 const (
-	maxRetryTimes    int = 3 // tikv-importer has done retry internally. so we don't retry many times.
-	retryBackoffTime     = time.Second * 3
+	maxRetryTimes           int = 3 // tikv-importer has done retry internally. so we don't retry many times.
```

Comment:
The type annotation `int` could be removed. And keep aligning with L38 will much pretty.